dashboard/db/schema.rb

```diff
@@ -1338,6 +1338,29 @@
     t.string   "emails"
   end
 
+  create_table "survey_answers", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+    t.integer  "user_id"
+    t.bigint   "form_id"
+    t.bigint   "submission_id"
+    t.string   "question_id"
+    t.text     "answer_value",  limit: 65535
+    t.datetime "created_at",                  null: false
+    t.datetime "updated_at",                  null: false
+  end
+
+  create_table "survey_questions", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
```

Comment:
I understand that this is our current design in which we separate submissions (with answers) from questions. My main concern is answers are fixed while questions could change over time. An answer submitted a month ago might be for a different question than the one we download today even though they have the same question id.